modules/custom/moj_resources/src/CategoryFeaturedContentApiClass.php

```diff
@@ -83,53 +83,53 @@ protected function translateNode(NodeInterface $node)
     return $node->hasTranslation($this->lang) ? $node->getTranslation($this->lang) : $node;
   }
   /**
-   * Get nids
+   * Get node ids
    *
    * @return void
    */
   protected function getFeaturedContentNodeIds($category, $number, $prison = 0)
   {
     $series = $this->promotedSeries($category, $prison);
     $nodes = $this->promotedNodes($category, $number, $prison);
-    $results = array_merge($series, $nodes);
+    $query = array_merge($series, $nodes);
```

Comment:
...and this actually is results and not a Drupal query. The queries are within the `promoted<blah>` functions. Can go through it with you tomorrow if you like?